src/mod/say/mod_say_nb/mod_say_nb.c

```diff
@@ -83,8 +83,8 @@ static switch_status_t play_group(switch_say_method_t method, switch_say_gender_
 			if ( a > 1 && b == 0 && c == 0) {	/* [2-9]00 */
 				switch_say_file(sh, "digits/%d", a); //to till ni
 			}
+            // File is unavailable but will probably not be needed
```

Comment:
interesting indent.

Reply:
Yeah, I don't know what is up with this formatting (and neither does VSCode), but it's like 2 tabs worth of indentation with 4 spaces per tab or something.

Reply:
Also: "looks fine on my machine"
![image](https://user-images.githubusercontent.com/43165886/198042356-0635d0ae-fc9e-4b49-8c9c-f93d36535080.png)